stylelint-plugin/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
issue: this isn't a standalone package, so it shouldn't have its own dependencies or commands.

Move the commands to the root package.json, and install the devDependencies there, if needed